plotille/_figure.py

```diff
@@ -28,6 +28,7 @@
 from itertools import cycle
 import os
 
+import numpy as np
```

Comment:
This is not a dependency of `plotille` and i would like to keep it that way. Please implement with stdlib funktions.